app/forms/file_yourself_form.rb

```diff
@@ -0,0 +1,20 @@
+class FileYourselfForm < Form
+  include FormAttributes
+  attr_accessor :diy_intake
+  set_attributes_for :diy_intake, :email_address, :preferred_first_name, :received_1099, :filing_frequency, :referrer, :locale, :visitor_id, :source
+
+  validates :email_address, 'valid_email_2/email': true, presence: true
+  validates :preferred_first_name, presence: true
+  validates :received_1099, presence: true
+  # TODO: Validate that this check works i.e. Rails sends empty string
```

Comment:
i think the works but maybe we wanted to add another test and i can't remember what it was

Reply:
One idea I had was manually testing this.

Reply:
I manually tested this, and it works great, so I'll delete the TODO.